docs/quickstart.rst

```diff
@@ -5,7 +5,7 @@ Quickstart
 
 In this section, we provide a simple guide for running a Kafka cluster along with all of the other Confluent Platform components.  By the end of this quickstart, you will have successfully installed and run a simple deployment including each component with Docker.
 
-In order to keep things simple, this quickstart guide is limited to a single node Confluent Platform cluster.  For more advanced tutorials, you can refer to the `tutorials section <tutorials.html>`_ of this documentation.
+In order to keep things simple, this quickstart guide is limited to a single node Confluent Platform cluster.  For more advanced tutorials, you can refer to the `tutorials section <tutorials/tutorials.html>`_ of this documentation.
```

Comment:
need to use RST anchors, e.g.

```
:ref:`tutorials_overview`
```

although you also need to fix this (extra space after anchor and `:`): https://github.com/confluentinc/cp-docker-images/blame/3.1.0-post/docs/tutorials/tutorials.rst#L1